### PR TITLE
[FRONT-264] Replace telegram and dev forum links with the Discord link

### PR DESCRIPTION
**Pull request by mondoreale**
Things I took care of

- Top nav, both desktop and mobile, both "guest" menu (docs) and the user menu (when signed in)
- Animated icons in the footer
- Docs + reindexing + 1 updated screenshot

Hope that's a complete list. :)

cc @fonty1 @mattatgit

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -118,7 +118,7 @@
     "@storybook/react": "^5.3.21",
     "@streamr/po-loader": "^1.0.0",
     "@streamr/streamr-icons": "^0.1.9",
-    "@streamr/streamr-layout": "^2.4.9",
+    "@streamr/streamr-layout": "^2.4.10",
```
Comment (mondoreale) [inline app/package.json:121]: New version has that one new Discord icon in it.

File: app/src/shared/i18n/index.js [hunk 2]
```diff
@@ -1,13 +1,5 @@
-// @flow
-
 import en from './en.po'
-import de from './de.po'
```
Comment (mondoreale) [inline app/src/shared/i18n/index.js:4]: As a side effect of this PR, I also dropped non-en translations. Seems only reasonable at this point.

**Comment by mondoreale** [comment]
Rebased. FYI.

**Comment by mondoreale** [comment]
I'll go ahead and merge this in and let's see it on staging.

**Comment by fonty1** [comment]
@mondoreale landing + platform look 👌 on staging.

**Pull request merged**